UI/UnoSQLiteOneDriveInvoiceSample/Demo/Demo/Demo.Shared/App.xaml.cs

```diff
@@ -99,7 +99,7 @@ protected override void OnLaunched(LaunchActivatedEventArgs args)
             }
 
 #if !(NET5_0 && WINDOWS)
```

Comment:
@aesteves900 this needs to be changed to net5 or greater otherwise you end up with a crash on Windows